core/src/main/resources/db/cgds.sql

```diff
@@ -279,15 +278,31 @@ CREATE TABLE `genetic_profile_samples` (
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE
 );
 
+CREATE TABLE `gene_panel` (
+    `INTERNAL_ID` int(11) NOT NULL auto_increment,
+    `STABLE_ID` varchar(255) NOT NULL,
+    `DESCRIPTION` mediumtext,
+    PRIMARY KEY (`INTERNAL_ID`),
+    UNIQUE (`STABLE_ID`)
+);
+CREATE TABLE `gene_panel_list` (
+    `INTERNAL_ID` int(11) NOT NULL,
+    `GENE_ID` int(255) NOT NULL,
+    PRIMARY KEY (`INTERNAL_ID`, `GENE_ID`),
+    FOREIGN KEY (`INTERNAL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE,
+    FOREIGN KEY (`GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
+);
 --
 -- Table structure for table `sample_profile`
 --
 CREATE TABLE `sample_profile` (
   `SAMPLE_ID` int(11) NOT NULL,
   `GENETIC_PROFILE_ID` int(11) NOT NULL,
+  `PANEL_ID` int(11) DEFAULT NULL,
   UNIQUE KEY `UQ_SAMPLE_ID_GENETIC_PROFILE_ID` (`SAMPLE_ID`,`GENETIC_PROFILE_ID`), -- Constraint to allow each sample only once in each profile
   FOREIGN KEY (`GENETIC_PROFILE_ID`) REFERENCES `genetic_profile` (`GENETIC_PROFILE_ID`) ON DELETE CASCADE,
-  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE
+  FOREIGN KEY (`SAMPLE_ID`) REFERENCES `sample` (`INTERNAL_ID`) ON DELETE CASCADE,
+  FOREIGN KEY (`PANEL_ID`) REFERENCES `gene_panel` (`INTERNAL_ID`) ON DELETE CASCADE
```

Comment:
@sheridancbio Not all genetic profiles are annotated with gene panel information. In fact, in existing data files, no genetic profile is. In those cases, a sample-profile relation (which is what a record in the `sample_profile` table represents) does not have an associated gene panel.

As an alternative to the nullable foreign key, this could be represented as a link table between the `sample_profile` table and the `gene_panel` table: a table with a foreign key to the primary key of each of the two tables. That would require adding a primary key constraint to the existing table `sample_profile`, and a unique constraint to the corresponding foreign key in the new link table to disallow many-to-many relations.